### ISSUE TITLE
Do we already have argb hex string converter?

### ISSUE THREAD

**Issue by kof** [MoOx/color#72]
Like this one http://lesscss.org/functions/#color-definition-argb

**Comment by MoOx** [comment]
Nope, but if something needs to be implemented, it will be probably `#rrggbbaa`.
Should be done easily by adding `hexAlphaString` in `color-string` package.

**Comment by kof** [comment]
Nope, its AARRGGBB

**Comment by kof** [comment]
https://msdn.microsoft.com/en-us/library/ms532930(v=vs.85).aspx

> Color is expressed in #AARRGGBB format, where AA is the alpha hexadecimal value, RR is the red hexadecimal value, GG is the green hexadecimal value, and BB is the blue hexadecimal value. The alpha value controls the opacity of the object. An alpha value of 00 is transparent, while a value of FF is opaque.

**Comment by Qix-** [comment]
ARGB or RGBA are two different representations - both are valid, but some are used in different places. RGBA makes more sense from an API standpoint since common overloading practices append "optional" parameters to the end.

**Comment by kof** [comment]
yep, however what I was asking for was specifically a string for IE compatible argb thing.

**Comment by Qix-** [comment]
Unless I'm mistaken, you should be able to pass ARGB values to RGBA and get the intended result.

**Comment by kof** [comment]
Well I need to covert to ARGB, not from.

**Comment by kof** [comment]
I didn't find any reference whether IE filter supports rgba or anything else.

**Comment by MoOx** [comment]
w3c css color spec only mention #rrggbbaa (or #rgba) https://drafts.csswg.org/css-color/#hex-notation

Anyway, PR are always welcome. I think we can bring that with alphaHexString (#argb) and hexAlphaString (#rgba).

**Comment by kof** [comment]
I think its IE only non standard thing.

**Comment by Qix-** [comment]
(*sigh*) Of course it's an IE thing. I'll add this in the new version.

**Comment by kof** [comment]
:+1:

**Comment by MoOx** [comment]
@Qix- It's in recent CSS specs too.

**Comment by Qix-** [comment]
This is actually a color-string addition :) I'll create a ticket for you.

Thanks for bringing it up!

**Comment by hoangsonbk** [comment]
Hi, just to check if we have this feature already?

**Comment by Qix-** [comment]
Nope, feel free to help out and open a PR :)

**Comment by n0ruSh** [comment]
I have opened a [PR](https://github.com/Qix-/color/pull/237) for this. Please help review :- @Qix-
